scripts/get-gee-build-environment.sh

```diff
@@ -32,14 +32,14 @@ MIN_GCC_VERSION="4.8"
 : ${PACKAGE_BUILD_DIR:=""}
```

Comment:
This file has been removed from this repository.  This change should not be part of this pull request.